src/elements/content-sidebar/activity-feed/activity-feed/ActivityThread.scss

```diff
@@ -36,3 +44,50 @@
         outline: auto;
     }
 }
+
+.be .bcs-activity-feed .bcs-activity-feed-items-container .bcs-ActivityItem.bcs-is-focused .bcs-ActivityThread {
+    padding-bottom: 0;
+    box-shadow: 0 1px $white;
+
+    &,
+    &:hover {
+        background-color: $white;
+    }
+
+    .bcs-ActivityThread-selectWrapper {
+        background-color: $bdl-light-blue-05;
+        border: $sidebarActivityFeedSelectedItemBorderWidth $bdl-box-blue solid;
+        border-radius: 6px;
```

Comment:
Can we use any existing variables for this value?

Reply:
Right, I've missed the `$bdl-border-radius` variables.